stories/core/Footer.stories.tsx

```diff
@@ -9,6 +9,10 @@ import { Footer, FooterProps } from '../../src/core';
 export default {
   title: 'Core/Footer',
   component: Footer,
+  argTypes: {
```

Comment:
Didnt you tell me in other PR not to use argTypes...?

Reply:
I said not to use `argTypes.defaultValue` 😄

Reply:
yeah right, double standards..